### DISCUSSION TITLE
Paraview

### DISCUSSION THREAD

**Discussion by HaodeW** [Anura3D/Anura3D_OpenSource#23] [General]

**Comment by murphy-john** [comment]
Hi HaodeW, This appears to be an issue with Paraview and not with Anura3D. Will you post this question to a Paraview forum and remove it from here? We'd like to keep this discussion thread focused on Anura3D.

**Reply by HaodeW** [reply]
I have posted this question to a Paraview forum prior but to no avail, which is why I thought maybe some Anura3D users can share their experience. For some reason (1st time using this), I am not able to find the delete button anywhere, so i removed the content instead. Hopefully that will suffice.

**Reply by murphy-john** [reply]
Hi HaodeW,
It is okay, I will try to help and I can remove the comment once we resolve the issue.

What version of Paraview are you using? I recall a number of our developers having trouble with the newest versions of Paraview. I use Paraview 5.9.0.

**Reply by HaodeW** [reply]
I have not tried 5.9.0. I followed the tutorial from Anura3D and used v5.0.1. I also attempted the newest version 5.10. Both versions crashed upon opening the software and I am not sure how to figure out the problem with it. Though I would not need paraview if I am able to somehow visualize the results manually though I am still figuring out the process for it because thus far, I have been only able to obtain the results from material points (via text/excel) which doesnt require paraview. Thank you!

**Reply by murphy-john** [reply]
Can you refresh my memory about the issue you're having? What graphics emulator are you using?

**Reply by HaodeW** [reply]
Hi John, basically the issue is that Paraview crashes instantly as soon as I open the software regardless of the version. The computer only has a microsoft basic display adaptor, which i gathered doesnt qualify as a graphics card but rather uses the CPU to create graphics. Though I need a more concrete idea that it is the graphics card issue which causes the crash since this is part of school property. I would also appreciate if you know of any other methods to extract/visualize results without paraview. Thus far, I have only been able to extract results via text for designated material points. Thank you!

**Reply by murphy-john** [reply]
Hello, We output material point and mesh data at each loadstep to the VTK file format. You could look for other visualization software which can read VTK files.
